src/Game.jsx

```diff
@@ -71,8 +76,8 @@ export default function Game() {
             clearInterval(id);
         }
     }
-    // event handlers ---------------------------------------------------------------
 
+    // event handlers ---------------------------------------------------------------
```

Comment:
If you do want to use comments to segregate common codes it would be better to have wrap the code in comments. Something like this:
```
// Event handlers ------------------------------
function event() {
  // event logic
}

function event2() {
  // event logic
}
// End of event handlers ------------------------------
```

But I would still suggest you not to use comments to describe code sections. You code should be self explanatory in most cases.

Reply:
extracted functions to appropriate files and made code self explanatory